examples/helloworld_orion/config.xml

```diff
@@ -19,7 +19,7 @@
 		<!-- <transformToNGSIv2>C:\dev\ros2\src\ros2\firos2\examples\helloworld_orion\build\Release\userlib.dll</transformToNGSIv2>
 		<bridge_library>C:\dev\ros2\build\firos2\Release\isrtpsngsiv2bridgelib.dll</bridge_library> -->
 		<!-- Linux -->
-		<transformToNGSIv2>/home/luisgp/ros2_ws/src/ros2/firos2/examples/helloworld_orion/build/libuserlib.so</transformToNGSIv2>
+		<transformToNGSIv2>~/ros2_ws/src/ros2/firos2/examples/helloworld_orion/build/libuserlib.so</transformToNGSIv2>
```

Comment:
If the example makes the execution path fixed, this path could be relative to it.
There are some examples in Integration Service.
In this case, if the example must be executed from `<ROS2Path>/src/ros2/firos2/examples/helloworld_orion`, the path  can be simplyfied to `<transformToNGSIv2>build/libuserlib.so</transformToNGSIv2>`.